### PR TITLE
[expo-document-picker][android] Fix file uri

### LINKED ISSUE #13627: Expo SDK 42: Expo Document picker does not return correct uri
### Summary

I am using the expo document picker to let the user select a pdf file and send it to the server. 

It was working previously when I was running on expo SDK 41 until I upgrade it to Expo SDK 42.

Before this, it was returning a uri starting with file:///data/ where now, it returns a uri starting with /data. 

I tried to append the uri to include file:// at the front but its not working either.

Previous version:
- "expo": "~41.0.1",
- "expo-document-picker": "~9.1.2",

Current version:
- "expo": "^42.0.0",
- "expo-document-picker": "~9.2.4",

### Managed or bare workflow? If you have `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

Android, iOS

### SDK Version (managed workflow only)

42

### Environment

  Expo CLI 4.7.3 environment info:
    System:
      OS: Windows 10 10.0.19042
    Binaries:
      Node: 14.17.1 - C:\Program Files\nodejs\node.EXE
      npm: 6.14.13 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: Version  4.2.0.0 AI-202.7660.26.42.7486908
    npmPackages:
      expo: ^42.0.0 => 42.0.1
      react: 16.13.1 => 16.13.1
      react-dom: 16.13.1 => 16.13.1
      react-native: https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz => 0.63.2
      react-native-web: ~0.13.12 => 0.13.18
    Expo Workflow: managed

### Reproducible demo or steps to reproduce from a blank project

see snack expo 42 (incorrect uri): https://snack.expo.io/@syazwan.dev/sdk42-document-picker

see snack expo 41 (expected uri): https://snack.expo.io/@syazwan.dev/sdk41-document-picker

### PR DESCRIPTION
**Pull request by mstach60161**
# Why

- Fixes #13627.
- File uri didn't have prefix "file://".

# How

- Added file prefix to output file path.

# Test Plan

- Native component list.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.io and README.md).

### CODE REVIEW

File: packages/expo-document-picker/android/src/main/java/expo/modules/documentpicker/DocumentPickerModule.kt [hunk 1]
```diff
@@ -132,6 +132,6 @@ class DocumentPickerModule(
       return null
     }
 
-    return outputFilePath
+    return "file://$outputFilePath"
```
Comment (lukmccall) [inline packages/expo-document-picker/android/src/main/java/expo/modules/documentpicker/DocumentPickerModule.kt:135]:
```suggestion
    return "file://$outputFilePath"
```
Are we sure if the `outputFilePath` won't contain a file scheme even if someone using a custom document picker? 
I think no. We need to wrap it in something which will be a little more intelligent. Maybe something like:
https://github.com/expo/expo/blob/22b3ce370fa3848e1046177201e5daa57ca2bc09/packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/tasks/ImageResultTask.kt#L36

**Review by lukmccall — CHANGES_REQUESTED** [review]

**Comment by pedrohenriquebraga** [comment]
Merge this with urgency

**Comment by adanielyan** [comment]
Is there any workaround to this issue until the fix is merged?

**Comment by lukmccall** [comment]
@adanielyan, you can add the 'file' scheme manually if not exists.

**Review by lukmccall — APPROVED** [review]
LGTM ✅

**Comment by syazwancode** [comment]
> @adanielyan, you can add the 'file' scheme manually if not exists.

Hi im not sure appending the 'file' manually will fix this issue. I have mention in the issue that I raised (https://github.com/expo/expo/issues/13627) that appending the uri to include file:// at the front does not work.

**Comment by adanielyan** [comment]
> > @adanielyan, you can add the 'file' scheme manually if not exists.
> 
> Hi im not sure appending the 'file' manually will fix this issue. I have mention in the issue that I raised (#13627) that appending the uri to include file:// at the front does not work.

Indeed, I tried making changes from this commit manually on the file in node_modules, but if didn't fix the issue. Had to rollback to sdk 41.

**Pull request merged**